src/models/manager.js

```diff
@@ -81,7 +81,11 @@ class manager {
                     .getMembers(channelId)
                     .then((members) => {
                         meeting.setMembers(members);
-
+                        channel
+                            .getChannelInfo(channelId)
+                            .then((info) => {
+                                meeting.setName(info.name)
```

Comment:
getChannelInfo function is a async function. you should call meeting.start function after channel info is requested.